modules/nw-route-specific-annotations.adoc

```diff
@@ -54,3 +55,33 @@ metadata:
 Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
+
+
+.A route rewrite-target annotation
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/rewrite-target: / <1>
+...
+----
+<1> Sets / as rewrite path of the request on the backend
+
```

Comment:
A paragraph explaining the annotation and the table below would be helpful.  For example,

> Setting the `haproxy.router.openshift.io/rewrite-target` annotation on a route specifies that the Ingress Controller should rewrite paths in HTTP requests using this route before forwarding the requests to the backend application.  The part of the request path that matches the path specified in `spec.path` is replaced with the rewrite target specified in the annotation.  The following table provides examples of the path rewriting behavior for various combinations of `spec.path`, request path, and rewrite target.

Reply:
paragraph added 👍